server/service/src/login.rs

```diff
@@ -229,6 +229,16 @@ impl LoginService {
                 "" => None,
                 _ => Some(user_info.user.e_mail.to_string()),
             },
+            language: match user_info.user.language {
```

Comment:
I _really_  wanted to make an enum instead but couldn't see how to make it work proper

Reply:
Would be like this one: https://github.com/openmsupply/open-msupply/compare/731-per-user-language...731-language-enum

At this point on front end would need to do some mapping from the enum.

I like enums, although I am getting somewhat convinced that in some cases they are not ideal, but for me it's similar to abbreviations issue (or escaping rust/db keywords, rather then re-naming things to avoid conflict):
* Better not to abbreviate to reduce ambiguity (I think all agree)
* Ohh but in some cases we can abbreviate when it's obvious (or when non abbreviated is too long) <- and here is where it all breaks for me, what is obvious ?, to whom? is it going to continue being obvious ? 

So basically, in my view, it's easier if we just draw the line (less discussion, don't need to think if particular abbreviation is obvious, etc..), never abbreviate or always enums if the variants are known at compile time. Anyways that's just my opinion.

Reply:
that's awesome! thanks - have added in e3b00dd6b and popped in the f/e mapping of Enum to strings